src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -21,6 +21,13 @@ object SaveMethod extends Enumeration {
   val Text = Value("text")
 }
 
+object DataframeSaveMode extends Enumeration {
```

Comment:
Tiny suggestion, this is already a defined class inside Spark so we could probably just use that instead instead of making a new class